build.gradle

```diff
@@ -58,6 +59,10 @@ dependencies {
     integrationTest24 jacksonCore + '2.4.0'
     integrationTest24 jacksonAnnotations + '2.4.0'
     integrationTest24 jacksonDatabind + '2.4.0'
+
+    integrationTest24 jacksonCore + '2.5.0'

```

Comment:
Maybe it's because you put `integrationTest24` here instead of `integrationTest25`?